pkg/api/ecosystem/restoreRestClient.go

```diff
@@ -0,0 +1,251 @@
+package ecosystem
```

Comment:
🚫 **[checkstyle]** <dupl> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>1-251 lines are duplicate of `pkg/api/ecosystem/backupRestClient.go:1-251`

Reply:
I'm inclined to ignore this.